src/cpu/x64/jit_avx512_core_bf16_conv_kernel.hpp

```diff
@@ -252,7 +256,7 @@ struct jit_avx512_core_bf16_fwd_kernel {
 
 private:
     DNNL_DISALLOW_COPY_AND_ASSIGN(jit_avx512_core_bf16_fwd_kernel);
```

Comment:
As noted in an earlier PR, this macro is redundant once we have a non-copyable `unique_ptr` as a member varaible.